modules/nrf_wifi/bus/qspi_if.c

```diff
@@ -195,7 +190,7 @@ struct qspi_nor_data {
 	struct k_sem sync;
 	/* The semaphore to control driver init/uninit. */
 	struct k_sem count;
-#else /* CONFIG_MULTITHREADING */
+#else  /* CONFIG_MULTITHREADING */
```

Comment:
Could you remove these unnecessary changes. I see it here and below in multiple places.